app/swagger/swagger/v1/requests/appeals/appeals.rb

```diff
@@ -259,4 +259,33 @@ class Swagger::V1::Requests::Appeals::Appeals
       end
     end
   end
+
+  swagger_path '/v1/decision_review_evidence' do
```

Comment:
this is the exact same as `app/swagger/swagger/requests/decision_review_evidence.rb` because v1 is the same as v0